Description: account for the "github.com/go-check/check" -> "gopkg.in/check.v1" rename
Author: Tianon Gravi <tianon@debian.org>
Forwarded: not-needed

diff --git a/integration-cli/check_test.go b/integration-cli/check_test.go
index defa02e..f9e2c67 100644
--- a/integration-cli/check_test.go
+++ b/integration-cli/check_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"testing"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func Test(t *testing.T) {
diff --git a/integration-cli/docker_api_attach_test.go b/integration-cli/docker_api_attach_test.go
index 06a1c48..001631e 100644
--- a/integration-cli/docker_api_attach_test.go
+++ b/integration-cli/docker_api_attach_test.go
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"golang.org/x/net/websocket"
 )
 
diff --git a/integration-cli/docker_api_containers_test.go b/integration-cli/docker_api_containers_test.go
index d8dc448..571af60 100644
--- a/integration-cli/docker_api_containers_test.go
+++ b/integration-cli/docker_api_containers_test.go
@@ -17,7 +17,7 @@ import (
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/pkg/stringid"
 	"github.com/docker/docker/runconfig"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestContainerApiGetAll(c *check.C) {
diff --git a/integration-cli/docker_api_events_test.go b/integration-cli/docker_api_events_test.go
index 8ae7bf7..9120aa0 100644
--- a/integration-cli/docker_api_events_test.go
+++ b/integration-cli/docker_api_events_test.go
@@ -4,7 +4,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestEventsApiEmptyOutput(c *check.C) {
diff --git a/integration-cli/docker_api_exec_resize_test.go b/integration-cli/docker_api_exec_resize_test.go
index 01061ca..817c734 100644
--- a/integration-cli/docker_api_exec_resize_test.go
+++ b/integration-cli/docker_api_exec_resize_test.go
@@ -4,7 +4,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestExecResizeApiHeightWidthNoInt(c *check.C) {
diff --git a/integration-cli/docker_api_exec_test.go b/integration-cli/docker_api_exec_test.go
index 3d99fe6..0446230 100644
--- a/integration-cli/docker_api_exec_test.go
+++ b/integration-cli/docker_api_exec_test.go
@@ -8,7 +8,7 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Regression test for #9414
diff --git a/integration-cli/docker_api_images_test.go b/integration-cli/docker_api_images_test.go
index 339a2f4..ab21d4b 100644
--- a/integration-cli/docker_api_images_test.go
+++ b/integration-cli/docker_api_images_test.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiImagesFilter(c *check.C) {
diff --git a/integration-cli/docker_api_info_test.go b/integration-cli/docker_api_info_test.go
index 4084289..1616e0e 100644
--- a/integration-cli/docker_api_info_test.go
+++ b/integration-cli/docker_api_info_test.go
@@ -4,7 +4,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInfoApi(c *check.C) {
diff --git a/integration-cli/docker_api_inspect_test.go b/integration-cli/docker_api_inspect_test.go
index 2287e7d..a57e687 100644
--- a/integration-cli/docker_api_inspect_test.go
+++ b/integration-cli/docker_api_inspect_test.go
@@ -6,7 +6,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
diff --git a/integration-cli/docker_api_logs_test.go b/integration-cli/docker_api_logs_test.go
index d478447..d22f363 100644
--- a/integration-cli/docker_api_logs_test.go
+++ b/integration-cli/docker_api_logs_test.go
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLogsApiWithStdout(c *check.C) {
diff --git a/integration-cli/docker_api_network_test.go b/integration-cli/docker_api_network_test.go
index 44d2b31..2da958f 100644
--- a/integration-cli/docker_api_network_test.go
+++ b/integration-cli/docker_api_network_test.go
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func isNetworkAvailable(c *check.C, name string) bool {
diff --git a/integration-cli/docker_api_resize_test.go b/integration-cli/docker_api_resize_test.go
index c7a577b..e421403 100644
--- a/integration-cli/docker_api_resize_test.go
+++ b/integration-cli/docker_api_resize_test.go
@@ -4,7 +4,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestResizeApiResponse(c *check.C) {
diff --git a/integration-cli/docker_api_service_test.go b/integration-cli/docker_api_service_test.go
index df07219..27b63c8 100644
--- a/integration-cli/docker_api_service_test.go
+++ b/integration-cli/docker_api_service_test.go
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func isServiceAvailable(c *check.C, name string, network string) bool {
diff --git a/integration-cli/docker_api_stats_test.go b/integration-cli/docker_api_stats_test.go
index f019e00..d1868fd 100644
--- a/integration-cli/docker_api_stats_test.go
+++ b/integration-cli/docker_api_stats_test.go
@@ -10,7 +10,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestCliStatsNoStreamGetCpu(c *check.C) {
diff --git a/integration-cli/docker_api_test.go b/integration-cli/docker_api_test.go
index 6cbf301..0572583 100644
--- a/integration-cli/docker_api_test.go
+++ b/integration-cli/docker_api_test.go
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/api"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestApiOptionsRoute(c *check.C) {
diff --git a/integration-cli/docker_api_version_test.go b/integration-cli/docker_api_version_test.go
index b756794..42fb872 100644
--- a/integration-cli/docker_api_version_test.go
+++ b/integration-cli/docker_api_version_test.go
@@ -6,7 +6,7 @@ import (
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/autogen/dockerversion"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestGetVersion(c *check.C) {
diff --git a/integration-cli/docker_cli_attach_test.go b/integration-cli/docker_cli_attach_test.go
index f45f775..90e389c 100644
--- a/integration-cli/docker_cli_attach_test.go
+++ b/integration-cli/docker_cli_attach_test.go
@@ -9,7 +9,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const attachWait = 5 * time.Second
diff --git a/integration-cli/docker_cli_attach_unix_test.go b/integration-cli/docker_cli_attach_unix_test.go
index 9718dc0..c30ff45 100644
--- a/integration-cli/docker_cli_attach_unix_test.go
+++ b/integration-cli/docker_cli_attach_unix_test.go
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_build_test.go b/integration-cli/docker_cli_build_test.go
index 6dc24df..ad56d5d 100644
--- a/integration-cli/docker_cli_build_test.go
+++ b/integration-cli/docker_cli_build_test.go
@@ -21,7 +21,7 @@ import (
 	"github.com/docker/docker/builder/command"
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/stringutils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestBuildJSONEmptyRun(c *check.C) {
diff --git a/integration-cli/docker_cli_build_unix_test.go b/integration-cli/docker_cli_build_unix_test.go
index fc7bd92..c246dbd 100644
--- a/integration-cli/docker_cli_build_unix_test.go
+++ b/integration-cli/docker_cli_build_unix_test.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/ulimit"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestBuildResourceConstraintsAreUsed(c *check.C) {
diff --git a/integration-cli/docker_cli_by_digest_test.go b/integration-cli/docker_cli_by_digest_test.go
index 71f8b1a..a53e3db 100644
--- a/integration-cli/docker_cli_by_digest_test.go
+++ b/integration-cli/docker_cli_by_digest_test.go
@@ -9,7 +9,7 @@ import (
 	"github.com/docker/distribution/digest"
 	"github.com/docker/distribution/manifest"
 	"github.com/docker/docker/utils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 var (
diff --git a/integration-cli/docker_cli_commit_test.go b/integration-cli/docker_cli_commit_test.go
index 125b2e3..7e0d6f5 100644
--- a/integration-cli/docker_cli_commit_test.go
+++ b/integration-cli/docker_cli_commit_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestCommitAfterContainerIsDone(c *check.C) {
diff --git a/integration-cli/docker_cli_config_test.go b/integration-cli/docker_cli_config_test.go
index 2b08f47..1b574fb 100644
--- a/integration-cli/docker_cli_config_test.go
+++ b/integration-cli/docker_cli_config_test.go
@@ -11,7 +11,7 @@ import (
 
 	"github.com/docker/docker/autogen/dockerversion"
 	"github.com/docker/docker/pkg/homedir"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestConfigHttpHeader(c *check.C) {
diff --git a/integration-cli/docker_cli_cp_from_container_test.go b/integration-cli/docker_cli_cp_from_container_test.go
index 945a34f..f6f3159 100644
--- a/integration-cli/docker_cli_cp_from_container_test.go
+++ b/integration-cli/docker_cli_cp_from_container_test.go
@@ -4,7 +4,7 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // docker cp CONTAINER:PATH LOCALPATH
diff --git a/integration-cli/docker_cli_cp_test.go b/integration-cli/docker_cli_cp_test.go
index 64ae0b5..94f6935 100644
--- a/integration-cli/docker_cli_cp_test.go
+++ b/integration-cli/docker_cli_cp_test.go
@@ -10,7 +10,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const (
diff --git a/integration-cli/docker_cli_cp_to_container_test.go b/integration-cli/docker_cli_cp_to_container_test.go
index 341121d..a705ad1 100644
--- a/integration-cli/docker_cli_cp_to_container_test.go
+++ b/integration-cli/docker_cli_cp_to_container_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"os"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // docker cp LOCALPATH CONTAINER:PATH
diff --git a/integration-cli/docker_cli_cp_utils.go b/integration-cli/docker_cli_cp_utils.go
index c26ebfd..9ed8a7d 100644
--- a/integration-cli/docker_cli_cp_utils.go
+++ b/integration-cli/docker_cli_cp_utils.go
@@ -10,7 +10,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/archive"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 type fileType uint32
diff --git a/integration-cli/docker_cli_create_test.go b/integration-cli/docker_cli_create_test.go
index 482e96f..9a0ad7e 100644
--- a/integration-cli/docker_cli_create_test.go
+++ b/integration-cli/docker_cli_create_test.go
@@ -13,7 +13,7 @@ import (
 	"io/ioutil"
 
 	"github.com/docker/docker/pkg/nat"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Make sure we can create a simple container with some args
diff --git a/integration-cli/docker_cli_daemon_experimental_test.go b/integration-cli/docker_cli_daemon_experimental_test.go
index dc4f792..51c51d2 100644
--- a/integration-cli/docker_cli_daemon_experimental_test.go
+++ b/integration-cli/docker_cli_daemon_experimental_test.go
@@ -6,7 +6,7 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func assertNetwork(c *check.C, d *Daemon, name string) {
diff --git a/integration-cli/docker_cli_daemon_test.go b/integration-cli/docker_cli_daemon_test.go
index 992cd83..2dec0ec 100644
--- a/integration-cli/docker_cli_daemon_test.go
+++ b/integration-cli/docker_cli_daemon_test.go
@@ -17,7 +17,7 @@ import (
 
 	"github.com/docker/libnetwork/iptables"
 	"github.com/docker/libtrust"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerDaemonSuite) TestDaemonRestartWithRunningContainersPorts(c *check.C) {
diff --git a/integration-cli/docker_cli_diff_test.go b/integration-cli/docker_cli_diff_test.go
index b5fc1bd..4041620 100644
--- a/integration-cli/docker_cli_diff_test.go
+++ b/integration-cli/docker_cli_diff_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // ensure that an added file shows up in docker diff
diff --git a/integration-cli/docker_cli_events_test.go b/integration-cli/docker_cli_events_test.go
index 6742ea4..2bcbfcd 100644
--- a/integration-cli/docker_cli_events_test.go
+++ b/integration-cli/docker_cli_events_test.go
@@ -13,7 +13,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestEventsTimestampFormats(c *check.C) {
diff --git a/integration-cli/docker_cli_events_unix_test.go b/integration-cli/docker_cli_events_unix_test.go
index 1a08f2b..fd6dcd6 100644
--- a/integration-cli/docker_cli_events_unix_test.go
+++ b/integration-cli/docker_cli_events_unix_test.go
@@ -10,7 +10,7 @@ import (
 	"os/exec"
 	"unicode"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_exec_test.go b/integration-cli/docker_cli_exec_test.go
index 8e85988..a626a7d 100644
--- a/integration-cli/docker_cli_exec_test.go
+++ b/integration-cli/docker_cli_exec_test.go
@@ -15,7 +15,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestExec(c *check.C) {
diff --git a/integration-cli/docker_cli_exec_unix_test.go b/integration-cli/docker_cli_exec_unix_test.go
index 28c202c..fa0b274 100644
--- a/integration-cli/docker_cli_exec_unix_test.go
+++ b/integration-cli/docker_cli_exec_unix_test.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_experimental_test.go b/integration-cli/docker_cli_experimental_test.go
index 694222b..100b6c4 100644
--- a/integration-cli/docker_cli_experimental_test.go
+++ b/integration-cli/docker_cli_experimental_test.go
@@ -5,7 +5,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestExperimentalVersion(c *check.C) {
diff --git a/integration-cli/docker_cli_export_import_test.go b/integration-cli/docker_cli_export_import_test.go
index a9e75de..f227755 100644
--- a/integration-cli/docker_cli_export_import_test.go
+++ b/integration-cli/docker_cli_export_import_test.go
@@ -5,7 +5,7 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // export an image and try to import it into a new one
diff --git a/integration-cli/docker_cli_help_test.go b/integration-cli/docker_cli_help_test.go
index 3113083..2c83671 100644
--- a/integration-cli/docker_cli_help_test.go
+++ b/integration-cli/docker_cli_help_test.go
@@ -8,7 +8,7 @@ import (
 	"unicode"
 
 	"github.com/docker/docker/pkg/homedir"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
diff --git a/integration-cli/docker_cli_history_test.go b/integration-cli/docker_cli_history_test.go
index 355e4c8..7faac52 100644
--- a/integration-cli/docker_cli_history_test.go
+++ b/integration-cli/docker_cli_history_test.go
@@ -6,7 +6,7 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // This is a heisen-test.  Because the created timestamp of images and the behavior of
diff --git a/integration-cli/docker_cli_images_test.go b/integration-cli/docker_cli_images_test.go
index fe12f02..1bc04ef 100644
--- a/integration-cli/docker_cli_images_test.go
+++ b/integration-cli/docker_cli_images_test.go
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestImagesEnsureImageIsListed(c *check.C) {
diff --git a/integration-cli/docker_cli_import_test.go b/integration-cli/docker_cli_import_test.go
index ccfc452..5d25068 100644
--- a/integration-cli/docker_cli_import_test.go
+++ b/integration-cli/docker_cli_import_test.go
@@ -7,7 +7,7 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestImportDisplay(c *check.C) {
diff --git a/integration-cli/docker_cli_info_test.go b/integration-cli/docker_cli_info_test.go
index 86719f4..134c5c6 100644
--- a/integration-cli/docker_cli_info_test.go
+++ b/integration-cli/docker_cli_info_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/utils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // ensure docker info succeeds
diff --git a/integration-cli/docker_cli_inspect_experimental_test.go b/integration-cli/docker_cli_inspect_experimental_test.go
index fada86e..4c4f98b 100644
--- a/integration-cli/docker_cli_inspect_experimental_test.go
+++ b/integration-cli/docker_cli_inspect_experimental_test.go
@@ -4,7 +4,7 @@ package main
 
 import (
 	"github.com/docker/docker/api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInspectNamedMountPoint(c *check.C) {
diff --git a/integration-cli/docker_cli_inspect_test.go b/integration-cli/docker_cli_inspect_test.go
index 3e42d0c..c52c2c8 100644
--- a/integration-cli/docker_cli_inspect_test.go
+++ b/integration-cli/docker_cli_inspect_test.go
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestInspectImage(c *check.C) {
diff --git a/integration-cli/docker_cli_kill_test.go b/integration-cli/docker_cli_kill_test.go
index 685f4f5..a3989b0 100644
--- a/integration-cli/docker_cli_kill_test.go
+++ b/integration-cli/docker_cli_kill_test.go
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestKillContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_links_test.go b/integration-cli/docker_cli_links_test.go
index 568f20a..e76f83d 100644
--- a/integration-cli/docker_cli_links_test.go
+++ b/integration-cli/docker_cli_links_test.go
@@ -2,7 +2,7 @@ package main
 
 import (
 	"fmt"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"reflect"
 	"regexp"
 	"strings"
diff --git a/integration-cli/docker_cli_links_unix_test.go b/integration-cli/docker_cli_links_unix_test.go
index 67a4464..bb886df 100644
--- a/integration-cli/docker_cli_links_unix_test.go
+++ b/integration-cli/docker_cli_links_unix_test.go
@@ -7,7 +7,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLinksEtcHostsRegularFile(c *check.C) {
diff --git a/integration-cli/docker_cli_login_test.go b/integration-cli/docker_cli_login_test.go
index 3b4431d..409a007 100644
--- a/integration-cli/docker_cli_login_test.go
+++ b/integration-cli/docker_cli_login_test.go
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"os/exec"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestLoginWithoutTTY(c *check.C) {
diff --git a/integration-cli/docker_cli_logs_test.go b/integration-cli/docker_cli_logs_test.go
index 6c94217..076e227 100644
--- a/integration-cli/docker_cli_logs_test.go
+++ b/integration-cli/docker_cli_logs_test.go
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/timeutils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // This used to work, it test a log of PageSize-1 (gh#4851)
diff --git a/integration-cli/docker_cli_nat_test.go b/integration-cli/docker_cli_nat_test.go
index a0773fe..d1044f5 100644
--- a/integration-cli/docker_cli_nat_test.go
+++ b/integration-cli/docker_cli_nat_test.go
@@ -6,7 +6,7 @@ import (
 	"net"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func startServerContainer(c *check.C, msg string, port int) string {
diff --git a/integration-cli/docker_cli_network_test.go b/integration-cli/docker_cli_network_test.go
index 08b225d..d931add 100644
--- a/integration-cli/docker_cli_network_test.go
+++ b/integration-cli/docker_cli_network_test.go
@@ -5,7 +5,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func assertNwIsAvailable(c *check.C, name string) {
diff --git a/integration-cli/docker_cli_pause_test.go b/integration-cli/docker_cli_pause_test.go
index 4e32dfc..72c9b4c 100644
--- a/integration-cli/docker_cli_pause_test.go
+++ b/integration-cli/docker_cli_pause_test.go
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPause(c *check.C) {
diff --git a/integration-cli/docker_cli_port_test.go b/integration-cli/docker_cli_port_test.go
index 63bfc9a..4eb5d44 100644
--- a/integration-cli/docker_cli_port_test.go
+++ b/integration-cli/docker_cli_port_test.go
@@ -6,7 +6,7 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPortList(c *check.C) {
diff --git a/integration-cli/docker_cli_port_unix_test.go b/integration-cli/docker_cli_port_unix_test.go
index 0988ca9..4cbf0ee 100644
--- a/integration-cli/docker_cli_port_unix_test.go
+++ b/integration-cli/docker_cli_port_unix_test.go
@@ -6,7 +6,7 @@ import (
 	"net"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPortHostBinding(c *check.C) {
diff --git a/integration-cli/docker_cli_proxy_test.go b/integration-cli/docker_cli_proxy_test.go
index 8b55c67..45737f0 100644
--- a/integration-cli/docker_cli_proxy_test.go
+++ b/integration-cli/docker_cli_proxy_test.go
@@ -5,7 +5,7 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestCliProxyDisableProxyUnixSock(c *check.C) {
diff --git a/integration-cli/docker_cli_ps_test.go b/integration-cli/docker_cli_ps_test.go
index 3b8f6f9..25a2e87 100644
--- a/integration-cli/docker_cli_ps_test.go
+++ b/integration-cli/docker_cli_ps_test.go
@@ -11,7 +11,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestPsListContainers(c *check.C) {
diff --git a/integration-cli/docker_cli_pull_test.go b/integration-cli/docker_cli_pull_test.go
index 8bfca8d..581366b 100644
--- a/integration-cli/docker_cli_pull_test.go
+++ b/integration-cli/docker_cli_pull_test.go
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	"github.com/docker/distribution/digest"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // See issue docker/docker#8141
diff --git a/integration-cli/docker_cli_push_test.go b/integration-cli/docker_cli_push_test.go
index 111e9f3..a087258 100644
--- a/integration-cli/docker_cli_push_test.go
+++ b/integration-cli/docker_cli_push_test.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Pushing an image to a private registry.
diff --git a/integration-cli/docker_cli_rename_test.go b/integration-cli/docker_cli_rename_test.go
index cac9f3a..3ecc814 100644
--- a/integration-cli/docker_cli_rename_test.go
+++ b/integration-cli/docker_cli_rename_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/stringid"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRenameStoppedContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_restart_test.go b/integration-cli/docker_cli_restart_test.go
index 4cabeb9..c558c3d 100644
--- a/integration-cli/docker_cli_restart_test.go
+++ b/integration-cli/docker_cli_restart_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRestartStoppedContainer(c *check.C) {
diff --git a/integration-cli/docker_cli_rm_test.go b/integration-cli/docker_cli_rm_test.go
index 0e57551..69104bf 100644
--- a/integration-cli/docker_cli_rm_test.go
+++ b/integration-cli/docker_cli_rm_test.go
@@ -4,7 +4,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
diff --git a/integration-cli/docker_cli_rmi_test.go b/integration-cli/docker_cli_rmi_test.go
index 8d9f94f..48faf9c 100644
--- a/integration-cli/docker_cli_rmi_test.go
+++ b/integration-cli/docker_cli_rmi_test.go
@@ -5,7 +5,7 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
diff --git a/integration-cli/docker_cli_run_test.go b/integration-cli/docker_cli_run_test.go
index 9288957..a04d03f 100644
--- a/integration-cli/docker_cli_run_test.go
+++ b/integration-cli/docker_cli_run_test.go
@@ -20,7 +20,7 @@ import (
 
 	"github.com/docker/docker/pkg/nat"
 	"github.com/docker/libnetwork/resolvconf"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // "test123" should be printed by docker run
diff --git a/integration-cli/docker_cli_run_unix_test.go b/integration-cli/docker_cli_run_unix_test.go
index 295bace..140b2ba 100644
--- a/integration-cli/docker_cli_run_unix_test.go
+++ b/integration-cli/docker_cli_run_unix_test.go
@@ -14,7 +14,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/mount"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_save_load_test.go b/integration-cli/docker_cli_save_load_test.go
index 5b4b64d..eb0d63f 100644
--- a/integration-cli/docker_cli_save_load_test.go
+++ b/integration-cli/docker_cli_save_load_test.go
@@ -10,7 +10,7 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // save a repo using gz compression and try to load it using stdout
diff --git a/integration-cli/docker_cli_save_load_unix_test.go b/integration-cli/docker_cli_save_load_unix_test.go
index 2bca3b8..991b940 100644
--- a/integration-cli/docker_cli_save_load_unix_test.go
+++ b/integration-cli/docker_cli_save_load_unix_test.go
@@ -8,7 +8,7 @@ import (
 	"os"
 	"os/exec"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 	"github.com/kr/pty"
 )
 
diff --git a/integration-cli/docker_cli_search_test.go b/integration-cli/docker_cli_search_test.go
index d89c05c..40db0df 100644
--- a/integration-cli/docker_cli_search_test.go
+++ b/integration-cli/docker_cli_search_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // search for repos named  "registry" on the central registry
diff --git a/integration-cli/docker_cli_service_test.go b/integration-cli/docker_cli_service_test.go
index aaf5e81..91b7b8b 100644
--- a/integration-cli/docker_cli_service_test.go
+++ b/integration-cli/docker_cli_service_test.go
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func assertSrvIsAvailable(c *check.C, sname, name string) {
diff --git a/integration-cli/docker_cli_start_test.go b/integration-cli/docker_cli_start_test.go
index ce5c48e..8027735 100644
--- a/integration-cli/docker_cli_start_test.go
+++ b/integration-cli/docker_cli_start_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Regression test for https://github.com/docker/docker/issues/7843
diff --git a/integration-cli/docker_cli_start_volume_driver_unix_test.go b/integration-cli/docker_cli_start_volume_driver_unix_test.go
index 71f9f2d..1fddee0 100644
--- a/integration-cli/docker_cli_start_volume_driver_unix_test.go
+++ b/integration-cli/docker_cli_start_volume_driver_unix_test.go
@@ -12,7 +12,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func init() {
diff --git a/integration-cli/docker_cli_stats_test.go b/integration-cli/docker_cli_stats_test.go
index 2504028..7cd6367 100644
--- a/integration-cli/docker_cli_stats_test.go
+++ b/integration-cli/docker_cli_stats_test.go
@@ -5,7 +5,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestCliStatsNoStream(c *check.C) {
diff --git a/integration-cli/docker_cli_tag_test.go b/integration-cli/docker_cli_tag_test.go
index 23f2aef..109dae3 100644
--- a/integration-cli/docker_cli_tag_test.go
+++ b/integration-cli/docker_cli_tag_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 
 	"github.com/docker/docker/pkg/stringutils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // tagging a named image in a new unprefixed repo should work
diff --git a/integration-cli/docker_cli_top_test.go b/integration-cli/docker_cli_top_test.go
index 667a6c8..cbca969 100644
--- a/integration-cli/docker_cli_top_test.go
+++ b/integration-cli/docker_cli_top_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func (s *DockerSuite) TestTopMultipleArgs(c *check.C) {
diff --git a/integration-cli/docker_cli_v2_only.go b/integration-cli/docker_cli_v2_only.go
index aa3d6a2..539f459 100644
--- a/integration-cli/docker_cli_v2_only.go
+++ b/integration-cli/docker_cli_v2_only.go
@@ -6,7 +6,7 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 func makefile(contents string) (string, func(), error) {
diff --git a/integration-cli/docker_cli_version_test.go b/integration-cli/docker_cli_version_test.go
index f2d8b65..b1ea96c 100644
--- a/integration-cli/docker_cli_version_test.go
+++ b/integration-cli/docker_cli_version_test.go
@@ -3,7 +3,7 @@ package main
 import (
 	"strings"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // ensure docker version works
diff --git a/integration-cli/docker_cli_wait_test.go b/integration-cli/docker_cli_wait_test.go
index 167ea1a..0503fbb 100644
--- a/integration-cli/docker_cli_wait_test.go
+++ b/integration-cli/docker_cli_wait_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // non-blocking wait with 0 exit code
diff --git a/integration-cli/docker_utils.go b/integration-cli/docker_utils.go
index e80015d..fcb3242 100644
--- a/integration-cli/docker_utils.go
+++ b/integration-cli/docker_utils.go
@@ -24,7 +24,7 @@ import (
 	"github.com/docker/docker/opts"
 	"github.com/docker/docker/pkg/ioutils"
 	"github.com/docker/docker/pkg/stringutils"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 // Daemon represents a Docker daemon for the testing framework.
diff --git a/integration-cli/registry.go b/integration-cli/registry.go
index 35e1b4e..4aee021 100644
--- a/integration-cli/registry.go
+++ b/integration-cli/registry.go
@@ -9,7 +9,7 @@ import (
 	"path/filepath"
 
 	"github.com/docker/distribution/digest"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 const v2binary = "registry-v2"
diff --git a/integration-cli/registry_mock.go b/integration-cli/registry_mock.go
index e5fb64c..9599eb2 100644
--- a/integration-cli/registry_mock.go
+++ b/integration-cli/registry_mock.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 type handlerFunc func(w http.ResponseWriter, r *http.Request)
diff --git a/integration-cli/requirements.go b/integration-cli/requirements.go
index ce080d5..a08febe 100644
--- a/integration-cli/requirements.go
+++ b/integration-cli/requirements.go
@@ -10,7 +10,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 type testCondition func() bool
diff --git a/integration-cli/trust_server.go b/integration-cli/trust_server.go
index 89d88a8..dcd245d 100644
--- a/integration-cli/trust_server.go
+++ b/integration-cli/trust_server.go
@@ -12,7 +12,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/pkg/tlsconfig"
-	"github.com/go-check/check"
+	"gopkg.in/check.v1"
 )
 
 var notaryBinary = "notary-server"
